build.go

```diff
@@ -901,3 +915,71 @@ func exitStatus(err error) int {
 
 	return -1
 }
+
+type gometalinter struct {
+	deadlineInSeconds int
+	command           string
+	linter            string
+	params            []string
+	directories       []string
+	excludes          []string
+}
+
+func NewGometalinter(linter string, dirs ...string) *gometalinter {
+	return &gometalinter{
+		deadlineInSeconds: 60,
+		command:           "gometalinter",
+		linter:            linter,
+		directories:       dirs,
+	}
+}
+
+func (g *gometalinter) verifyInstalled() {
+	if _, err := runError(g.command, "--disable-all"); err != nil {
+		log.Fatalf("gometalinter is not installed")
```

Comment:
If this is fatal, let's check this externally rather than in run potentially 3 times.